scripts/lib/upgrade-zulip-stage-2

```diff
@@ -84,6 +84,9 @@ else:
     subprocess.check_call(["cp", "-rT", os.path.join(deploy_path, 'prod-static/serve'),
                            '/home/zulip/prod-static'], preexec_fn=su_to_zulip)
 
+migrations_output = subprocess.check_output(["./manage.py", "create_large_indexes"],
+                                            preexec_fn=su_to_zulip)
```

Comment:
Just a quick note here; `subprocess.check_all` was what you wanted here, since we don't want to capture the output, we want to be passed through.